packages/extension/src/gui.view.ts

```diff
@@ -162,15 +163,19 @@ export class MarqueeGui extends EventEmitter {
          * the extension properly exports a setup method
          */
         extension.exports &&
-        extension.exports.marquee &&
-        typeof extension.exports.marquee.setup === 'function'
-      ) {
-        const defaultState = extension.exports.marquee?.disposable?.state || {}
-        const defaultConfiguration = extension.exports.marquee?.disposable?.configuration || {}
-        const ch = new Channel(extension.id, { ...defaultState, ...defaultConfiguration })
-        ch.registerPromise([this.panel.webview]).then((client) => {
-          extension.exports.marquee.setup(client)
-        })
+        extension.exports.marquee
+      ) {       
+        if (typeof extension.exports.marquee.widgetCounter === 'number'){
+          customWidgetCounter += extension.exports.marquee.widgetCounter
+        }
```

Comment:
Why can't we assume that if nothing is set the extension exports 1 custom widget?

Reply:
Let me double check this with the stateful extension also running. but when I did the `|| 1` it counted the core widgets too?

Reply:
@christian-bromann inside the `gui.view.ts` is there a way to tell if the widget is a custom widget or a core widget?

Reply:
nvm I got it [...vscode.extensions.all]